File: app/src/main/java/com/hqq/example/app/dialog/FoldFullBottomSheet.java
package com.hqq.example.app.dialog;

import androidx.recyclerview.widget.LinearLayoutManager;
import androidx.recyclerview.widget.RecyclerView;

import com.hqq.example.ui.BaseBottomDialog;
import com.hqq.example.utils.ResourcesUtils;
import com.hqq.example.app.R;
import com.hqq.example.app.adapter.StringAdapter;

/**
 * @Author : huangqiqiang
 * @Package : com.hqq.coreapp.dialog
 * @FileName :   FullBottomSheet
 * @Date : 2019/4/29 0029  下午 2:06
 * @Email : qiqiang213@gmail.com
 * @Descrive :
 */
public class FoldFullBottomSheet extends BaseBottomDialog {
    StringAdapter mStringAdapter = new StringAdapter();

    @Override
    public int setViewId() {
        return R.layout.dialog_bottom_sheet;

    }

    @Override
    public void initView() {
        RecyclerView recyclerView = mRootView.findViewById(R.id.rc_list);
        recyclerView.setLayoutManager(new LinearLayoutManager(getContext()));
        for (int i = 0; i < 20; i++) {
            mStringAdapter.addData("    " + i);
        }
        recyclerView.setAdapter(mStringAdapter);
    }

    @Override
    public int setHeight() {
        return (int) ResourcesUtils.getDimen(getContext(), R.dimen.x750);
    }

    @Override
    protected int getTransparentBottomSheetStyle() {
        return R.style.TransparentBottomSheetStyle_behavior;
    }
}

File: app/src/main/java/com/hqq/example/dialog/FoldFullBottomSheet.kt
package com.hqq.example.dialog

import androidx.recyclerview.widget.LinearLayoutManager
import androidx.recyclerview.widget.RecyclerView
import com.hqq.core.ui.base.BaseBottomDialog
import com.hqq.example.R
import com.hqq.example.adapter.StringAdapter

/**
 * @Author : huangqiqiang
 * @Package : com.hqq.coreapp.dialog
 * @FileName :   FullBottomSheet
 * @Date : 2019/4/29 0029  下午 2:06
 * @Email : qiqiang213@gmail.com
 * @Descrive :
 */
class FoldFullBottomSheet : BaseBottomDialog() {
    var mStringAdapter = StringAdapter()
    override val layoutViewId: Int
        get() = R.layout.dialog_bottom_sheet

    override fun initView() {
        val recyclerView: RecyclerView = mRootView!!.findViewById(R.id.rc_list)
        recyclerView.layoutManager = LinearLayoutManager(context)
        for (i in 0..19) {
            mStringAdapter.addData("    $i")
        }
        recyclerView.adapter = mStringAdapter
    }

    override val transparentBottomSheetStyle: Int
        get() = R.style.TransparentBottomSheetStyle_behavior
}